pkgs/top-level/release-haskell.nix

```diff
@@ -0,0 +1,209 @@
+/*
+  To debug this expression you can use `hydra-eval-jobs` from
+  `pkgs.hydra-unstable` which prints the jobset description
+  to `stdout`:
+
+  $ hydra-eval-jobs -I . pkgs/top-level/release-haskell.nix
+*/
+{ supportedSystems ? [ "x86_64-linux" ] }:
```

Comment:
Shouldn't this be the same as with the other release jobs? I am thinking that at least `aarch64-linux` should be in here as well.

Reply:
This is like this for historical reasons. Since haskell packages get built on `aarch64-linux` and `x86_64-darwin` in the main jobsets as well, I think we can enable it here as well. I just want to poke the infra team if it's okay.